### DISCUSSION TITLE
MSMITE%40CMU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#277] [Announcements]
# MSMITE%40CMU/

None

https://opencs.app/MSMITE%40CMU/

**Comment by XXXJumpingFrogXXX** [comment]
后面的同学再申可以关注一下Advanced和Standard的区别 这个项目的Advanced好像没办法项目结项 只可以论文结项

**Reply by Ex10si0n** [reply]
还可以选多36学分的课

**Reply by XXXJumpingFrogXXX** [reply]
求给个官方说明网址
